c_api/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
```

Comment:
As this is a C project, needing a C compiler, we should declare a new `project()` here with `LANGUAGES C`.